guidelines/index.html

```diff
@@ -78,1254 +106,1196 @@ <h3>Section status levels</h3>
+						</ol>
+					</p>
 
-			<section>
-				<h3>Help and feedback</h3>
-				<section class="guideline"  data-status="exploratory">
-					<h4>Consistent help</h4>
-					<p class="guideline-text">Help is labeled consistently and provided in a consistent visual and programmatic location.</p>
-					<p class="ednote">Needs <a href="#additional_research">additional research</a></p>
-				</section>
-				<section class="guideline"  data-status="exploratory">
-					<h4>Contextual help</h4>
-					<p class="guideline-text">Contextual help is available.</p>
-				</section>
-				<section class="guideline"  data-status="exploratory">
-					<h4>Conversational support</h4>
-					<p class="guideline-text">Conversational support is available that allows both text and verbal communication.</p>
-				</section>
-				<section class="guideline"  data-status="exploratory">
-					<h4>Data visualization help</h4>
-					<p class="guideline-text">Help understanding and using data visualizations is available.</p>
-					<p class="ednote">Needs <a href="#additional_research">additional research</a></p>
-				</section>		
-				<section class="guideline"  data-status="exploratory">
-					<h4>Feedback mechanism</h4>
-					<p class="guideline-text">Feedback can be provided.</p>
-				</section>	
-				<section class="guideline"  data-status="exploratory">
-					<h4>Help using new interfaces</h4>
-					<p class="guideline-text">Help using new or changed interfaces is available.</p>
-					<p class="ednote">Needs <a href="#additional_research">additional research</a></p>
-				</section>
-				<section class="guideline"  data-status="exploratory">
-					<h4>Personalizable help</h4>
-					<p class="guideline-text">Adaptable/personalizable help is available.</p>
-					<p class="ednote">Needs <a href="#additional_research">additional research</a></p>
-				</section>
-				<section class="guideline"  data-status="exploratory">
-					<h4>Sensory characteristics</h4>
-					<p class="guideline-text">Instructions and help do not rely on sensory characteristics.</p>
-				</section>
-				<section class="guideline"  data-status="exploratory">
-					<h4>Supplements to text</h4>
-					<p class="guideline-text">Visual illustrations, pictures, and images are available to help explain complex ideas, events, and processes.</p>
-					<p class="ednote">Needs <a href="#additional_research">additional research</a></p>
-				</section>
-				<section class="guideline"  data-status="exploratory">
-					<h4>Support available</h4>
-					<p class="guideline-text">Accessible support during data entry, task completion and search is available.</p>
-					<p class="ednote">Needs <a href="#additional_research">additional research</a></p>
-				</section>		
-			</section>	
+					<section class="provision" data-status="developing">
+						<h5>Custom indicator</h5>
+						<p class="provision-text">A custom focus indicator is used with sufficient size, change of contrast, adjacent contrast, distinct style and adjacecy. </p>
```

Comment:
Typo?

```suggestion
						<p class="provision-text">A custom focus indicator is used with sufficient size, change of contrast, adjacent contrast, distinct style and adjacency. </p>
```

Reply:
I am also confused by what it means for a focus indicator to have sufficient adjacency, is this clear enough?

Reply:
There is (or will be) more in the how-to docs, I can't think of a better word for it though. The idea is that the focus indicator must be closer to the focused item than any other interactive item.

Reply:
Aaah I see! Agree that's a sensible requirement now that I understand it. Wonder if it's even helpful to write what you wrote as its own sentence?

```suggestion
						<p class="provision-text">A custom focus indicator is used with sufficient size, change of contrast, adjacent contrast and distinct style. It is also closer to the focused item than to any other interactive element.</p>
```